static/js/pivotal.js

```diff
@@ -176,18 +232,18 @@
         (function(project) {
```

Comment:
Do you know why we need this immediately invoked function ? 
Usually it's used to encapsulate some behaviour, but here I'm not sure why we have it.